### PR TITLE
Update step-7.php

### PR DESCRIPTION
**Pull request by SabineSIL**
Line 23: There is no entry F5 in the menu in Keyman-Developer -> Keyboard -> Test Keyboard.   "Test Keyboard F5"  would be nice

### CODE REVIEW

**Comment by mcdurdin** [comment]
There's no change made here. F5 does work, even if it isn't shown.

**Comment by SabineSIL** [comment]
Inside the menu of Keyman-Developer -> Keyboard  we have the line "Test Keyboard"  It would be nicer for the user to have  "**Test Keyboard     F5**" instead. ( As it is done for "Compile Keyboard    F7" or "Install...  Ctrl+F9" )

**Comment by mcdurdin** [comment]
Ah, I see. The <kbd>F5</kbd> shortcut is used with `Start Debugging` in the `Debug` menu instead.

File: developer/15.0/guides/develop/tutorial/step-7.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 <h2>Compiling the Keyboard</h2>
 
 <p>After compiling the keyboard successfully, we can start testing it. Choose <span class="guimenu">Keyboard</span>, <span class=
```
Comment (mcdurdin) [inline developer/15.0/guides/develop/tutorial/step-7.php:22]:
```suggestion
<p>After compiling the keyboard successfully, we can start testing it. Choose <span class="guimenu">Debug</span>, <span class=
```

File: developer/15.0/guides/develop/tutorial/step-7.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
...
-"guimenuitem">Start Test</span> or press <kbd>F5</kbd> to begin testing. The Keyboard test window will appear.</p>
+"guimenuitem">Start Test</span> or press <kbd>F5</kbd> to begin testing. The Keyboard test window will appear.</p>
```
Comment (mcdurdin) [inline developer/15.0/guides/develop/tutorial/step-7.php:23]: "guimenuitem">Start Debugging</span> or press <kbd>F5</kbd> to begin testing. The Keyboard debug window will appear.</p>

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**